packages/store-sync/src/storageAdapter100.bench.ts

```diff
@@ -0,0 +1,26 @@
+import { bench, describe } from "vitest";
+import worldRpcLogs from "../../../test-data/world-logs-100.json";
+import { recsStorageAdapter, sqliteStorageAdapter, zustandStorageAdapter } from "../test/utils";
+import { logsToBlocks } from "../test/logsToBlocks";
+
+const blocks = logsToBlocks(worldRpcLogs.slice(0, 100));
```

Comment:
This is not completely accurate because the first dozen records will be logs for the World internals, which are processed differently.